inaugurator/main.py

```diff
@@ -32,6 +32,7 @@
 parser.add_argument("--inauguratorDownload", nargs='+', default=[])
 parser.add_argument("--inauguratorIgnoreDirs", nargs='+', default=[])
 parser.add_argument("--inauguratorTargetDeviceCandidate", nargs='+')
+parser.add_argument("--inauguratorTargetDeviceLabel", default="BOOT")
```

Comment:
This should not have a default value. I think that the user should explicitly use one of the 3 flags needed (and if not, ceremony.py will throw an exception).